docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -40,8 +41,27 @@ \subsection{Problem}
 
 \subsection{Inputs and Outputs}
 
-\wss{Characterize the problem in terms of ``high level'' inputs and outputs.  
-Use abstraction so that you can avoid details.}
+\subsubsection{Inputs}
+    \begin{itemize}
+        \item Personal consumer spending data
+        \item Location data
+        \item User Preferences
+        \item Historical Shopping Lists
+        \item User Reviews and Ratings
+        \item Shopping History
+    \end{itemize}
+
+\subsubsection{Outputs}
+    \begin{itemize}
+        \item Analysis of spending data
+        \item Purchase suggestions based on spending data
+        \item Budgeting Plans
+        \item Local Price Comparison
+        \item Shopping Trip Summaries
+        \item Community Forums
+        \item Inventory Alerts
+        \item Environmental Impact Information
+    \end{itemize}
```

Comment:
Community Forums, Inventory Alerts, and Environmental Impact Information are not really things we discussed for this app. Once again, I think they _could_ be stretch goals but I don't think they should be included here. I also think that in the context of the app, Purchase Suggestions and Local Price Comparison are somewhat repetitive since the suggestions we provide will be if someone bought something similar at a cheaper price elsewhere. If anything, I think you remove Local Price Comparison since that falls under Purchase Suggestions in our context. I'm also a bit uncertain about Budgeting Plans as something we discussed about in our initial scope. The app was initially described as allowing you to view trends with your own spending but I'm not sure we're implementing methods to budget within the app.

@allanfang1 @Tanger71 Thoughts?

Reply:
Removed Community Forums, Inventory Alerts, and Environmental Impact Information